src/util/Logger.js

```diff
@@ -10,18 +10,25 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+let debug = false;
+
 function log(level, args) {
   // Only log statements in development mode or if
   // throwing an error through console.error
-  if (DEBUG || level === 'error') {
+  if (debug || DEBUG || level === 'error') {
```

Comment:
todo: use `window.OktaSignInDebug.isEnabled()` instead of local `debug`?
And remove `setDebug()`

Reply:
Or rename `debug` options in SIW to `verbose` which is more appropriate name